src/core_atmosphere/physics/mpas_atmphys_driver_radiation_sw.F

```diff
@@ -949,13 +950,34 @@ subroutine driver_radiation_sw(itimestep,configs,mesh,state,time_lev,diag_physic
 
  call mpas_pool_get_config(configs,'config_o3climatology' ,config_o3climatology)
  call mpas_pool_get_config(configs,'config_radt_sw_scheme',radt_sw_scheme      )
+ call mpas_pool_get_config(configs,'config_dt'            ,dt                  )
 
  xtime_m = xtime_s/60.
 
+ if (itimestep == 1 .and. dt_radtsw == dt .and. &
+     (mpas_cpl%role_is(ROLE_INTEGRATE) .or. mpas_cpl%role_is(ROLE_RADIATION))) then
+     ! When running lagged radiation with dt == radt, the 2nd and 3rd timesteps would receive
+     ! SW radiation computed at 1 radt behind.
+     call mpas_log_write('')
+     call mpas_log_write('WARNING: Lagged radiation with dt==radt is not fully supported.')
```

Comment:
You can get a "proper" warning message by changing this line to:
```fortran
call mpas_log_write('Lagged radiation with dt==radt is not fully supported.', messageType=MPAS_LOG_WARN)
```
Using `messageType` will ensure that the warning message is counted towards the total number of warning messages printed at the end of the log file, and it will ensure that -- if we make adjustments to the style of warning messages -- this warning will always be consistent in style with other warning messages.

Reply:
Thank you for the suggestion. Missed this one last time I looked at the PR. I've pushed a new commit to implement this suggestion as well as a prior commit to address the other issue raised.